<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Animeplex - Login Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="LoginWindow_MainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>901</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	border-image: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="LoginWindow_BackBtnFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QPushButton" name="LoginWindow_BackBtn">
          <property name="minimumSize">
           <size>
            <width>95</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>95</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: white;	
	border: none;
}

QPushButton::hover {
	background-color: rgb(186, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>â®œ   BACK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="LoginWindow_TextFrame">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>551</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QLabel" name="LoginWindow_HeadingLabel">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>32</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: white;
}</string>
          </property>
          <property name="text">
           <string>Login To Account.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LoginWindow_TextLabel">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: white;
}</string>
          </property>
          <property name="text">
           <string>Enter in an account's email and password to login and proceed to the hub screen.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="LoginWindow_InputsFrame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>591</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="LoginWindow_FieldLabel_1">
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color: white;
}</string>
            </property>
            <property name="text">
             <string>Account Email Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LoginWindow_EmailLineEdit">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(27, 27, 27);
	color: white;
	border: none;
}</string>
            </property>
            <property name="placeholderText">
             <string>Ex: burner@gmail.com</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="LoginWindow_FieldLabel_2">
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>13</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color: white;
}</string>
            </property>
            <property name="text">
             <string>Account Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="LoginWindow_PasswordLineEdit">
            <property name="minimumSize">
             <size>
              <width>340</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(27, 27, 27);
	color: white;
	border: none;
}</string>
            </property>
            <property name="placeholderText">
             <string>Ex: password123</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="LoginWindow_BtnsFrame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>620</y>
       <width>591</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_20">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_12">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="LoginWindow_ClearBtn">
          <property name="minimumSize">
           <size>
            <width>145</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: white;	
	border: none;
}

QPushButton::hover {
	background-color: rgb(186, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>CLEAR INPUTS</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="LoginWindow_LoginBtn">
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(18, 18, 18);
	color: white;
	border: none;
}

QPushButton::hover {
	background-color: rgb(140, 4, 24);
}</string>
          </property>
          <property name="text">
           <string>LOGIN USER</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="LoginWindow_AccountDoesNotExistLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>785</y>
       <width>571</width>
       <height>0</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(204, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Error: That account does not exist.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="LoginWindow_EmptyInputsLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>780</y>
       <width>571</width>
       <height>0</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(204, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Error: Empty inputs</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="LoginWindow_SpacesDetectedLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>780</y>
       <width>571</width>
       <height>0</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: rgb(204, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Error: Spaces detected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
